src/coreclr/vm/appdomain.hpp

```diff
@@ -2394,7 +2394,7 @@ class SystemDomain : public BaseDomain
     //****************************************************************************************
     //
     // Global Static to get the one and only system domain
-    static SystemDomain * System()
+    static PTR_SystemDomain System()
```

Comment:
None of the methods is DACized.  Any reason you are singling this one out?

Reply:
This is called in SystemDomain::GetGlobalLoaderAllocator() which is called by BaseDomain::GetLoaderAllocator() which is called by ClrDataAccess::GetAppDomain() which was failing in the [commit](bc78804f5d1486bb7313d1661992626640091bf1) the vendors reported the cross DAC failures.  Since then (in main) it has been fixed.  There were some HOST_WINDOWS ifdef's (now removed) in BaseDomain which caused the field layout to be different in the Windows cross DAC and the Linux runtime/DAC. 

This change is left over from diagnosing the issue.  I left it because m_pSystemDomain is PTR_SystemDomain and is still used by the DAC.